test/index.ts

```diff
@@ -0,0 +1,22 @@
+// This file is loaded automatically by mocha during the test run.
+import { isWindows } from './utils/helper';
+import { execSync } from 'child_process';
+
+// Error code returned if we cannot even start testing:
+const PRETEST_ERR_RC = 2;
+
+if (isWindows()) {
```

Comment:
Can you please add a comment stating `isWindows` will return false while running on windows + wsl2 combination

Reply:
done, added comment and more verbose error message.